src/pages/components/Nav/Nav.js

```diff
@@ -1,14 +1,78 @@
 import React, { Component } from "react";
+import MenuDropBox from "./MenuDropBox/MenuDropBox";
 import "./Nav.scss";
 
 class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      menus: [],
+      hoveredIdx: ""
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/nav.json", { method: "GET" })
+      .then(nav => nav.json())
+      .then(nav => {
+        this.setState({
+          menus: nav.menus
+        });
```

Comment:
```suggestion
      .then(res => res.json())
      .then(({ menus }) => this.setState({ menus }));
```
- nav 보다는 res 등으로 받는 것이 보다 해당 변수의 역할을 설명하는데 적합할 것 같습니다
- 함수 파라미터 자리에서 직접 구조분해 할당을 해줄 수도 있습니다
- 객체의 키, 값 자리에 들어오는 변수명이 같을 경우 한쪽을 생략해서 써줄 수도 있습니다.

Reply:
오! 네 앞으로 구조분해 할당을 기억할게요~ 수정했습니다.